tests/input_output/output_tabular/enum.txt

```diff
@@ -0,0 +1,20 @@
++-----------------+-----------------+
+| Enum Name       | Enum Values     |
++-----------------+-----------------+
+| "Status"        | ["Good", "Bad"] |
++-----------------+-----------------+
++---------------------------+
+| n                         |
++---------------------------+
+| (:l1 {s: "Status::Good"}) |
```

Comment:
We want it to print as a literal enum, not a string
`(:l1 {s: Status::Good})`

Reply:
Good catch!